sql/sql_yacc.yy

```diff
@@ -1912,6 +1912,7 @@ void warn_about_deprecated_national(THD *thd)
         ts_option_undo_buffer_size
         ts_option_wait
 	ts_option_encryption
```

Comment:
First two chunks indent the added clause correctly, and the third one with ts_option_encryption. Let's fix ts_option_encryption indentation

Reply:
done